pkgs/tools/system/rwc/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "rwc";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "leahneukirchen";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "sha256-axHBkrbLEcYygCDofhqfIeZ5pv1sR34I5UgFUwVb2rI=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    description = "Report when files are changed";
+    license = licenses.publicDomain;
+    platforms = platforms.all;
```

```suggestion
    platforms = platforms.unix;
    badPlatforms = platforms.darwin;
```

Darwin does not compile because of `inotify`.

---

Come to think of it, it should probably be `platforms.linux` since it's a Linux-specific API, actually.